int32 scan1_index                                # Index of scan1
int32 scan2_index                                # Index of scan2
geometry_msgs/Pose loop_closure_constraint       # relates scan1 to scan2
bool failed_to_converge                          # true if ICET registration solution diverges

